lib/matplotlib/_layoutgrid.py

```diff
@@ -178,12 +180,13 @@ def parent_constraints(self):
         # parent's left, the last column right equal to the
         # parent's right...
         parent = self.parent
-        if parent is None:
-            hc = [self.lefts[0] == 0,
-                  self.rights[-1] == 1,
+        if not isinstance(parent, LayoutGrid):
+            # specify a rectangle in figure co-ordinates
```

Comment:
```suggestion
            # specify a rectangle in figure coordinates
```

Reply:
Fixed, though its a private module, and I like the British spelling better (at least I don't add an umlaut!) ;-)